### PR TITLE
Nicole Sullivan Autobio

### PR DESCRIPTION
**Pull request by sullivannicole**
_How changes were made to README.md_

1. Forked repository hw01 from uc-cfss on GitHub
2. Copied forked repository URL using green "Clone repository" button on GitHub
3. Created new project in RStudio using that repository URL (File>New Project>Version Control>Git; paste repository URL into "Repository URL" line; check "Open in new session"; click "Create Project")
4. Checked box next to "README.md" file (appears in the "Files" tab of the lower right pane)
5. Installed RMarkdown (code is:  install.packages("rmarkdown"))
6. Used online sources to write code in "README.md" for HW01
7. Used "Preview" button to view changes
8. Committed changes, pushed to GitHub
9. Refreshed GitHub repository to ensure changes appeared correctly

_Difficulties encountered_

While (of course) it took some experimentation to write the code correctly and preview it, the main difficulty I encountered was an error message when I attempted to commit changes.  Sometimes RStudio would return the message **fatal error**- and fatal is a frightening word to see on my computer!  However, when I would click "commit" again, the correct message was returned ("insertion, deletion" etc.), and when I pushed the changes to GitHub, they appeared nearly exactly as I had seen in the "Preview" option of RStudio (although I didn't realize that option existed for a commit or two!).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,49 @@
-# hw01
-Repository for first CFSS homework assignment
+***
+
+###Me,
```
Comment (nnickels) [inline README.md:3]: 1. Good use of headers!

File: README.md [hunk 2]
```diff
@@ -1,2 +1,49 @@
...
+####*short & sweet*   
+
+Hi!  I'm Nicole Sullivan! I'm in MAPSS,     
+studying social neuroscience,   
+and I love lightening up life with   
+(prepare for the cliche):
+
+***
+
+* **laughter**
```
Comment (nnickels) [inline README.md:13]: 2. Good use of bullet points!

File: README.md [hunk 3]
```diff
@@ -1,2 +1,49 @@
...
+
+![](http://2.bp.blogspot.com/-wdvOTaqFTgc/VRuiScblddI/AAAAAAAADsk/a3ZaYhlu35o/s1600/AIDSjoke.jpg)
```
Comment (nnickels) [inline README.md:15]: 3. Good use of images!

File: README.md (hunk 2 above)
Comment (nnickels) [inline README.md:13]: 4. Good use of bold!

File: README.md [hunk 4]
```diff
@@ -1,2 +1,49 @@
...
+*my mom*   
+
+![](http://static1.1.sqspcdn.com/static/f/804093/26459789/1439423910377/Brianna.jpg?token=I5s1RU7njJ7eSCS%2BwGK5cQtmyrk%3D)
+*my sister*   
+
+   ***
+   
+* **faith**
+
+> *There is a surely a future hope for you*
```
Comment (nnickels) [inline README.md:47]: Awesome block quote!

**Review by nnickels — COMMENTED** [review]
Evaluation for HW1 - sullivannicole reviewed by nnickels

Coding Style - NA
Coding Strategy - NA
Presentation - your presentation is lively! I could only view on Git and not in R (see explanation below). Looks good.
Achievement, creativity - excellent creativity! clear mastery of markdown, aside from not working when I clone your repo and open in R.
Ease of access - again, only issue was with reproducibility. 

- Your bio is great! So many images and great memes.
The first thing that I did was clone your repo from your Git homework page and open it in R. Although I could see your bio perfectly normally on Git (viewing the file as rich diff), when I open your markdown file in R and try to view the html, I get an error code for one of your images (pandoc: Could not fetch http://static1.1.sqspcdn.com/static/f/804093/26459745/1439422365683/Mark.jpg?token=bKWscXN%2BVTUq1yHEvhWE%2BxpghS0%3D) which keeps me from being able to view the html file. So I had to peer review just based on the bio viewable on Git.
- you really added a personal flair to the assignment!

**Comment by jmausolf** [comment]
## Evaluation HW-01 for Nicole Sullivan

| Topic | Excellent | Satisfactory | Needs Work |
| --- | --- | --- | --- |
| **Coding style** |        x  |     |             |
| **Coding strategy** |     x   |        |        |
| **Presentation: graphs** |      |     |       |
| **Presentation: tables** |       |       |     |
| **Achievement, creativity** |x      |      |         |
| **Ease of access** | x |                 |             |

Remarks:
- For the header of the assignment, you probably could have added your name at the top, instead of just "me," but I suppose your version also fits the style of the bio.
- You may want to add a .gitignore. There is actually a default one for R files that you can get when creating a new repo. For example, you could use this to avoid committing the .Rproj file. To use .gitignore: 

1. Commit the .gitignore before other commits
2. If you are using bash, you can run the command `git status` to check what files you have added before committing them. This is best practice, especially at first.

Full R .gitignore file (saved as `.gitignore`):
```
# History files
.Rhistory
.Rapp.history

# Session Data files
.RData

# Example code in package build process
*-Ex.R

# Output files from R CMD build
/*.tar.gz

# Output files from R CMD check
/*.Rcheck/

# RStudio files
.Rproj.user/

# produced vignettes
vignettes/*.html
vignettes/*.pdf

# OAuth2 token, see https://github.com/hadley/httr/releases/tag/v0.3
.httr-oauth

# knitr and R markdown default cache directories
/*_cache/
/cache/

# Temporary files created by R markdown
*.utf8.md
*.knit.md
```

- Otherwise, you effectively used four of the five style elements, italics, headers, list, and photos. You also used section breaks, which are nice. Good job.
- Great use of git. Proved you can do the workflow.

**Comment by xiaorancheng** [comment]
# Evaluation for Nicole Sullivan

Coding styleL: Excellent  
Coding strategy: Excellent  
Presentation(graphs): N/A  
Presentation(tables): N/A  
Achievement, creativity: Excellent  	
Ease of access: Excellent  

Nicole did a really good job. Her file included all the basic R Markdown technics mentioned on the course website. What's more, her work is very interesting. It showed great creativity. I like her content. I encountered fatal error as well. All in all, it's a great work!

**Pull request closed without merging**